src/executor/graph_executor.h

```diff
@@ -19,6 +19,8 @@
 #include <vector>
 #include "./exec_pass.h"
 
+#define EXECUTOR_DEBUG 0
```

Comment:
I'll consider change the DEBUG flag to run time environment variable and limit the debug message to only include the storage types of the node entries in the graph. This is something very useful when running a model and check what happens for each operator.